src/types/Workload/WorkloadComponent.tsx

```diff
@@ -0,0 +1,123 @@
+import Namespace from '../Namespace';
```

Comment:
This file shouldn't be under `src/types` I think. Basically, types shouldn't contain any TSX / rendering stuff. There's a `components` directory more suited for that.

Reply:
It's tax by the react component returned in the render method.